packages/markdown-docx/src/CiceroMarkToOOXML/index.js

```diff
@@ -36,11 +46,15 @@ const definedNodes = {
  */
 class CiceroMarkToOOXMLTransfomer {
     /**
-     * Declares the OOXML and counter variable.
+     * Declares the OOXML, counter and tags variable.
```

Comment:
`Declares the OOXML, counter, and tags variable.` Notice the comma after "counter".